openapi/components/schemas/Storefront/Account.yaml

```
@@ -43,3 +43,13 @@ properties:
     items:
       anyOf:
         - $ref: ../Links/SelfLink.yaml
+  _embedded:
+    type: array
+    description: >-
+      Any embedded objects available that are requested by the `expand`
+      querystring parameter.
```

nitpick: I'm not sure if we need to state "any" or "available" here

These words sound a bit passive 

```suggestion
      Available embedded objects that are requested by the `expand`
      querystring parameter.
```
or
```suggestion
      Embedded objects that are requested by the `expand`
      querystring parameter.
```